cmd/probe/internal/binding/mysql/mysql.go

```diff
@@ -176,7 +176,7 @@ func (mysqlOps *MysqlOperations) GetRoleForReplication(ctx context.Context, requ
 	return "", errors.Errorf("parse query failed, no records")
 }
 
-func (mysqlOps *MysqlOperations) GetRoleForConsensus(ctx context.Context, request *bindings.InvokeRequest, response *bindings.InvokeResponse) (string, error) {
+func (mysqlOps *MysqlOperations) GetRoleForConsensus(ctx context.Context) (string, error) {
```

Comment:
Add annotations and unittest for this function to make sure you can always get the correct role from any version of the target DB